merge_values_test.go

```diff
@@ -178,6 +184,16 @@ func Test_MergeValues(t *testing.T) {
 			},
 			expectedValues: map[string]interface{}{},
 		},
+		{
+			name: "case 10: wrong dest yaml returns error",
+			destMap: map[string][]byte{
+				"test": []byte(wrongYaml),
+			},
+			srcMap: map[string][]byte{
+				"test": []byte("test: val"),
+			},
+			errorMatcher: IsParsingDestFailedError,
```

Comment:
What's going to be error message we'll actually see in this case?

Reply:
It will be like this, ```{"kind":"parsingDestFailedError","annotation":"yaml: line 3: did not find expected key","stack":[{"file":"/Users/jihyukbok/go/src/github.com/giantswarm/helmclient/merge_values.go","line":27}```
If we could ensure that whether the destination or source file is a problem, then we can show this message to users by pointing the actual problem file.

Reply:
Higher level component making use of helm client (app/chart-operator?) could add context (what was destination, and what was source), to make error message more meaningful.

Reply:
Sure, I'm testing helmclient and it results like this. 
```
...
status:
  appVersion: ""
  release:
    lastDeployed: null
    reason: 'parsing error: failed to parse user configmap, logs from merging: parsing
      src failed error: yaml: line 22: did not find expected key'
    status: configmap-merge-failed
  version: ""
```